migration_steps/prepare_source_data/counts_verification/sql/count_non_cp1.sql

```diff
@@ -7,7 +7,7 @@ CREATE TABLE IF NOT EXISTS countverification.non_cp1_clients (id int);
 INSERT INTO countverification.non_cp1_clients (id) (
     SELECT id FROM persons p
     WHERE p.type = 'actor_client'
-      AND (caseactorgroup != 'CLIENT-PILOT-ONE' OR caseactorgroup IS NULL)
+      AND (COALESCE(caseactorgroup, '') != 'CLIENT-PILOT-ONE')
```

Comment:
This doesn't have logical difference. It's just to make it like the other script.